src/Servers/IIS/IIS/test/IIS.Http.Tests/IIS.Http.FunctionalTests.csproj

```diff
@@ -0,0 +1,59 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <TestGroupName>IIS.FunctionalTests</TestGroupName>
```

Comment:
This test group name needs updating, right?

Reply:
Why doesn't the directory name match the project name? IIS.Http.Tests vs IIS.Http.FunctionalTests

Reply:
I wasn't sure what we use test group names for at all now, is this a old parallelization thing?  It has no effect on helix since we only run on test project at a time, does it still affect things when we run build.cmd locally I guess?

Reply:
Yeah the directory name should match, thanks

Reply:
Right, it's used to limit parallelization across projects:
https://github.com/dotnet/aspnetcore/blob/52eff90fbcfca39b7eb58baad597df6a99a542b0/src/SignalR/Directory.Build.targets#L5-L6

I don't think you'd want any of these IIS tests running in parallel since they interact with the global IIS service.

Reply:
So we still need this but only for local runs?

Reply:
I think so